site-src/references/implementations.md

```diff
@@ -65,12 +65,16 @@ Gloo Edge 2.0 is an Istio-native, fully-featured Envoy based API gateway that br
 [Google Kubernetes Engine (GKE)][gke] is a managed Kubernetes platform
 offered by Google Cloud.
 
-GKE support for the Gateway API for orchestration of [Google Cloud Load
-Balancing][gclb] is available in [public preview][gke-gateway].
+GKE's implementation of the Gateway API is through the [GKE Gateway controller](gke-gateway) 
+(currently in Preview) which provisions [Google Cloud Load Balancers](gclb). 
+Deploy [private or public/internet Gateways](gke-gateway-deploy) or 
+[multi-cluster Gateways](gke-multi-cluster-gateways). 
 
 [gke]:https://cloud.google.com/kubernetes-engine
 [gclb]:https://cloud.google.com/load-balancing
 [gke-gateway]: https://cloud.google.com/kubernetes-engine/docs/concepts/gateway-api
+[gke-gateway-deploy]: https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-gateways
+[gke-multi-cluster-gateway]: https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-multi-cluster-gateways
```

Comment:
Testing out the links on the deploy preview makes it look like all three of these should not have a space after the colon.